.tekton/koku-metrics-operator-bundle-pull-request.yaml

```diff
@@ -7,8 +7,13 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "main"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull" && target_branch == "main" && 
+      (
+        ".tekton/koku-metrics-operator-bundle-pull.yaml".pathChanged() || 
+        "bundle/metadata/***".pathChanged() || 
+        "bundle/manifests/***".pathChanged() || 
+        "bundle.Dockerfile".pathChanged() 
+      )
```

Comment:
What's the purpose of changing the trigger for the pipeline? I usually see this used when setting up nudges, but I don't think we have components that need nudging.

Reply:
This pipeline doesn't need to run every time but only when bundle assets are updated. In the cluster service yaml in the manifests folder, we reference the [operator image](https://github.com/project-koku/koku-metrics-operator/blob/0ec274a9e1c7f3b581018dbf29870348156761a1/bundle/manifests/koku-metrics-operator.clusterserviceversion.yaml#L42) which will be replaced with the konflux-built image and once everything is configured, that's what will be nudged.